.travis.yml

```diff
@@ -5,15 +5,18 @@ branches:
 notifications:
         email: mongoose-im@erlang-solutions.com
 otp_release:
-        - 19.0
-        - 18.3
-        - 17.5
+        - 19.3
+        - 20.3
+        - 21.0
 install:
         - make
         - make test-compile
+        - travis_retry pip install --user codecov
 script:
         - make test
         - make dialyzer
+        - make codecov
```

Comment:
I think this makes sense only in case of `success` as the coverage report is sent only then.